docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,807 @@
+for the given example.
+
+### Step 3: Make sure the IP network is set up
+
+To follow the next steps, the IP network must be up and running. For instance,
+the Thread network can be established using
+[OpenThread Border Router](https://openthread.io/codelabs/openthread-border-router#0).
+
+### Step 4: Determine network pairing credentials
+
+You must provide the CHIP Tool with network credentials that will be used in the
+device commissioning procedure to configure the device with a network interface,
+such as Thread or Wi-Fi.
+
+The Matter specification does not define how the network credentials are to be
+obtained by controller.
+
+#### Thread network credentials
+
+Fetch and store the current Active Operational Dataset from the Thread Border
+Router. This step may vary depending on the Thread Border Router implementation.
+
+If you are using
+[OpenThread Border Router](https://openthread.io/codelabs/openthread-border-router#0)
+(OTBR), retrieve this information using one of the following commands:
+
+-   For OTBR running in Docker:
+
+    ```
+    sudo docker exec -it otbr sh -c "sudo ot-ctl dataset active -x"
+    0e080000000000010000000300001335060004001fffe002084fe76e9a8b5edaf50708fde46f999f0698e20510d47f5027a414ffeebaefa92285cc84fa030f4f70656e5468726561642d653439630102e49c0410b92f8c7fbb4f9f3e08492ee3915fbd2f0c0402a0fff8
+    Done
+    ```
+
+-   For OTBR native installation:
+
+    ```
+    sudo ot-ctl dataset active -x
+    0e080000000000010000000300001335060004001fffe002084fe76e9a8b5edaf50708fde46f999f0698e20510d47f5027a414ffeebaefa92285cc84fa030f4f70656e5468726561642d653439630102e49c0410b92f8c7fbb4f9f3e08492ee3915fbd2f0c0402a0fff8
+    Done
+    ```
+
+For Thread, you might also use a different out-of-band method to fetch the
+network credentials.
+
+#### Wi-Fi network credentials
+
+You must get the following Wi-Fi network credentials to commission the Matter
+device to the Wi-Fi network in the following steps:
+
+-   Wi-Fi SSID
+-   Wi-Fi password
+
+The steps required to determine the SSID and password may vary depending on the
+setup. For instance, you might need to contact your local Wi-Fi network
+administrator.
+
+### Step 5: Determine Matter device's _discriminator_ and _setup PIN code_
+
+Matter uses a 12-bit value called _discriminator_ to discern between multiple
+commissionable device advertisements and a 27-bit _setup PIN code_ to
+authenticate the device. You can find these values in the logging terminal of
+the device (for instance, UART). For example:
+
+```
+I: 254 [DL]Device Configuration:
+I: 257 [DL] Serial Number: TEST_SN
+I: 260 [DL] Vendor Id: 65521 (0xFFF1)
+I: 263 [DL] Product Id: 32768 (0x8000)
+I: 267 [DL] Hardware Version: 1
+I: 270 [DL] Setup Pin Code: 20202021
+I: 273 [DL] Setup Discriminator: 3840 (0xF00)
+I: 278 [DL] Manufacturing Date: (not set)
+I: 281 [DL] Device Type: 65535 (0xFFFF)
+```
+
+In above printout, the _discriminator_ is `3840 (0xF00)` and the _setup PIN
+code_ is equal to `20202021`.
```

Comment:
```suggestion

In this snippet, the discriminator is ``3840 (0xF00)`` and the setup PIN
code is equal to ``20202021``.
These values are automatically printed in the logging terminal for each device when the device boots up.
```

Reply:
But it's already mentioned above:

`You can find these values in the logging terminal of
the device (for instance, UART). For example:`

Reply:
OK, can you edit to something like:
`You can find these values in the logging terminal of the device (for instance, UART) when the device boots up. For example:`